Changes needed to get cppunit to build on Solaris.

These changes will be reported upstream.

--- cppunit-1.12.1/include/cppunit/portability/FloatingPoint.h.orig	2015-05-08 06:53:35.731634778 -0700
+++ cppunit-1.12.1/include/cppunit/portability/FloatingPoint.h	2015-05-08 06:53:16.614630891 -0700
@@ -3,6 +3,9 @@
 
 #include <cppunit/Portability.h>
 #include <math.h>
+#if defined(sun)
+#include <ieeefp.h>
+#endif
 
 CPPUNIT_NS_BEGIN
 
